### PR TITLE
Auto commit helm charts from kubermatic/kubermatic/config to kubermatic/kubermatic-installer/charts by tagging new version

### LINKED ISSUE #1076: Synchronize helm charts with kubermatic/kubermatic-installer
We want to automatically sync the current helm charts to kubermatic/kubermatic-installer.

### PR DESCRIPTION
**Pull request by cbrgm**
**What this PR does / why we need it**:
Deploy helm charts synchronously to dev & cloud

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1076

**Special notes for your reviewer**:
Needs to be tested.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -159,6 +159,21 @@ pipeline:
...
+  sync-charts:
+    image: alpine:3.7
+    pull: true
+    commands:
+      - apk add --no-cache -U git
+      - export $INSTALLER_DIR="/go/src/github.com/kubermatic/kubermatic-installer"
+      - git clone https://github.com/kubermatic/kubermatic-installer.git $INSTALLER_DIR && mkdir -p $INSTALLER_DIR/charts
+      - cp -r cert-manager certs kubermatic monitoring nginx-ingress-controller nodeport-proxy oauth $INSTALLER_DIR/charts
+      - cd $INSTALLER_DIR
+      - git add . && git commit -m "Synchronising helm charts from commit ${DRONE_COMMIT}" && git tag ${DRONE_TAG}
```
Comment (mrIncompetent) [inline .drone.yml]: no german

File: .drone.yml [hunk 2]
```diff
@@ -159,6 +159,21 @@ pipeline:
...
+      - git commit -m "Synchronising helm charts from commit $DRONE_COMMIT" && git push origin master --tags
```
Comment (mrIncompetent) [inline .drone.yml]: no german

**Comment by mrIncompetent** [comment]
PR title is wrong

**Comment by mrIncompetent** [comment]
Test before assigning someone. Unless its a one-line commit

**Comment by cbrgm** [comment]
What should the title look like @mrIncompetent ?

**Comment by mrIncompetent** [comment]
@cbrgm Probably similar to the issue this PR solves

**Comment by cbrgm** [comment]
`sync-charts` step is working now.

https://drone.loodse.com/kubermatic/kubermatic/1043/14
https://github.com/kubermatic/kubermatic-installer/tree/master/charts

File: .drone.yml [hunk 3]
```diff
@@ -159,6 +159,23 @@ pipeline:
...
+    pull: true
+    commands:
+      - apk add --no-cache -U git
+      - git config --global user.email "dev@loodse.com"
+      - git config --global user.name "drone"
+      - export INSTALLER_DIR="/go/src/github.com/kubermatic/kubermatic-installer"
+      - git clone https://github.com/kubermatic/kubermatic-installer.git $INSTALLER_DIR && mkdir -p $INSTALLER_DIR/charts
+      - cp -r config/cert-manager config/certs config/kubermatic config/monitoring config/nginx-ingress-controller config/nodeport-proxy config/oauth $INSTALLER_DIR/charts
+      - cd $INSTALLER_DIR
+      - git add . && git commit -m "Synchronising helm charts from commit ${DRONE_COMMIT}" && git tag ${DRONE_TAG}
```
Comment (metalmatze) [inline .drone.yml]: Can you put the `git tag ${DRONE_TAG}` in a new line?

**Review by metalmatze — CHANGES_REQUESTED** [review]
One nitpick, otherwise really solid! :100:

**Review by metalmatze — APPROVED** [review]
LGTM

**Review by mrIncompetent — APPROVED** [review]

**Pull request merged**